views/home.html

```diff
@@ -4,6 +4,26 @@
   <div class="row mx-3">
     <div class="col-md-6 col-12">
       <div class="row">
+         {{ if .IsClosed}}
+            <div class="alert alert-danger">
+                <h4 class="alert-heading mb-3">
+                    This Voting Service Provider is closed
+                </h4>
+                <p>
+                    {{ .ClosePoolMsg }}
+                </p>
+                <p>
+                    A closed VSP will still vote on tickets with already paid fees, but will not accept new any tickets.
```

Comment:
This isn't true for dcrstakepool - it will continue accepting tickets. This line can be removed.

Reply:
Okay, thanks.